tests/assets/config/valid.json

```diff
@@ -0,0 +1,19 @@
+
+{ 
+    "data": {
+        "grid": {
+            "name": "Daylight-Factor",
+            "object_name": "grid",
+            "delimiter": ",",
```

Comment:
Most of the files that are generated from honeybee recipes are tab / white space separated. I think it's more helpful to use an example like that instead of a modified file.

Reply:
Do you have samples I can use?

Reply:
I think this one requires a separate discussion and scoping.

Reply:
I don't think so. We already have sample files from Pollination - and none of them is comma separated. We should be able to handle them otherwise why are we developing this library?

Reply:
https://app.pollination.cloud/projects/devang/demo/jobs/e7b13422-38fe-476a-8514-79864c7f1d3c/runs/b8d67398-bc7c-5c8b-8639-87ced2d2aeae?path=%2Fmetrics&tab=workspace
I tried loading UDI results from above run and I was able to load it. I did not check other metrics but I did see those files. Their format is similar to UDI and therefore, I am confident we can load all these metrics. I don't know why we started this argument but the capability to load these files have been there for quite some time now. We have been loading daylight factor results which is the same file format.

Reply:
OK. Then we are good. I think my point was to use those files as samples instead of one with ',' in it which is not what our recipes generate right now.